enhancements/machine-config/machine-config-node.md

```diff
@@ -546,6 +593,217 @@ or
 The first option here indicates that this phase has never happened. The second one indicates that it has happened, just not during this update cycle. That is what the `Action during update to...` shows. That rendered config is not the one we are updating to currently.
 
+#### OnClusterLayering-Enabled Update Flow
+When OCL is enabled and a user triggers an OCL update, the update statuses should be reflected differently in the MCN resources to better match the steps taken in that update process. When an upgrade is triggered by there being a mismatch between a desired and current image or simply just a new MC or Containerfile change being applied, the MCNs for a specific pool will report the following phases. Note that, like in the previous update section, all phases will exist in the MCN object during the update, but the mermaid diagram below has been condensed to only highlight the conditions relevant to OCL-style updates.
+
+```mermaid
+block-beta
+  columns 16
+    block:parents:16
+        columns 16
+        parentTitle("Parent Phases")
+        space:17
+        updatePrepared["UpdatePrepared"] space:4
+        updateExecuted["UpdateExecuted"] space:2
+        rebootedNode["RebootedNode"] space
+        updateComplete["UpdateComplete"] space
+        updated["Updated"]
+        space:6
+        space:13
+    end
+
+    updatePrepared --> updateExecuted
+    updateExecuted --> rebootedNode
+    rebootedNode --> updateComplete
+    updateComplete --> updated
+
+  space:16
+
+    block:children:16
+        columns 16
+        childTitle("Child Phases")
+        space:16
+        imageBuilt["ImageBuilt"] space
+        imagePushedToRegistry["ImagePushedToRegistry"] space
+        imagePulledFromRegistry["ImagePulledFromRegistry"]
+        cordoned["Cordoned"]
+        drained["Drained"]
+        appliedOSImage["AppliedOSImage"] 
+        appliedFiles["AppliedFiles"] space:2
+        uncordoned["Uncordoned"] space:2
+        space:14
+    end
+
+    imageBuilt --> updatePrepared
+    imagePushedToRegistry --> updatePrepared
+    imagePulledFromRegistry --> updateExecuted
+    cordoned --> updateExecuted
+    drained --> updateExecuted
+    appliedOSImage --> updateExecuted
+    appliedFiles --> updateExecuted
+    uncordoned --> updateComplete
+
+class parents PhaseGroup
+class children PhaseGroup
+classDef PhaseGroup fill:#f5f5f5,stroke-dasharray:10,10,stroke-width:2px,stroke:#000
+
+class parentTitle PhaseTitle
+class childTitle PhaseTitle
+classDef PhaseTitle stroke:transparent,fill:transparent,font-weight:bold,font-size:1.25em,color:#000
+
+class updatePrepared,updateExecuted,imageBuilt,imagePushedToRegistry,imagePulledFromRegistry,updateComplete,updated,cordoned,drained,appliedOSImage,appliedFiles,rebootedNode,uncordoned Phase
+classDef Phase font-weight:bold,fill:#bbbbbb,stroke:#000,color:#000
+```
+
+*Before an update is triggered, UPDATED will be True and all other statuses will be False.*
+```console
+$ oc get machineconfignodes -o wide
```

Comment:
For ease of readability, could you maybe just list the output of the node undergoing the update here, and remove the other 5?